config/application.rb

```diff
@@ -24,6 +24,8 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.2
 
+    config.eager_load_paths << Rails.root.join("lib")
```

Comment:
One really fussy nitpick might be to just eager-load files under `lib/design_system` instead of `lib`

Reply:
Yeah agreed; though when I did this earlier it broke my name spacing, I think I would have to remove `DesignSystem::` from the namespaces for it to work that way, but I'll take another look 👍

Reply:
Ah, see your point - might take some lateral thinking